pep-0558.rst

```diff
@@ -35,6 +35,20 @@ It also proposes the addition of several supporting functions and type
 definitions to the CPython C API.
 
 
+Postponed to Python 3.10
+========================
```

Comment:
Not sure that this section is useful. Just update "Python-Version:" header, no?

This PEP missed Python 3.9 feature freeze.

Reply:
Fair point - I've taken it out.